src/library/playlisttablemodel.cpp

```diff
@@ -11,10 +11,10 @@
 PlaylistTableModel::PlaylistTableModel(QObject* parent,
                                        TrackCollectionManager* pTrackCollectionManager,
                                        const char* settingsNamespace,
-                                       bool showAll)
-        : BaseSqlTableModel(parent, pTrackCollectionManager, settingsNamespace),
+                                       bool preserveAll)
+        : TrackSetTableModel(parent, pTrackCollectionManager, settingsNamespace),
           m_iPlaylistId(-1),
-          m_showAll(showAll) {
+          m_preserveAll(preserveAll) {
```

Comment:
This was really unclear. 
How about: m_preserveHiddenTracks or invert m_removeHiddenTracks

Reply:
If we change what it does, we can just rename it then.

Reply:
fair point ^^